### ISSUE TITLE
Make Team Coffee Hour

### ISSUE THREAD

**Issue by eidolonnight** [WordPress/Marketing-Team#23]
This bi-weekly social hour is held by the Make Marketing team for all WordPress Marketing contributors, and occasional guests. Stop by and meet some of the friendly faces marketing WordPress.

Current volunteer facilitator list:
- 24 March 2022: Anjana Vasan
- 7 April 2022: Meg Phillips

**Comment by nalininsbs** [comment]
@eidolonnight is this now a coffee hour rather than titled a coffee break? It may give people an impression they have to stay or be available for an hour. If using Zoom, facilitators will need access to a pro zoom account to be able to run this for an hour. Forty minutes is the maximum for the free accounts. Thank you.

**Comment by nalininsbs** [comment]
Note: confirmed coffee break is scheduled for 40 minutes and not one hour. References to coffee hour are being updated by @eidolonnight.

**Comment by megphillips91** [comment]
Just asking that we update the project thread here to include the time in UTC. I keep forgetting - so sorry. If It is here I will not forget I promise. 

Would like to volunteer to host June 16th, 2022.

**Comment by murillol** [comment]
hi @megphillips91, I've just added the time to the first comment on the card :-)

**Comment by chaion07** [comment]
Hello! I volunteer to host the coffee break on 11 August 6:30 PM UTC. Kindly advise on the next steps. Thanks!

**Comment by DanSoschin** [comment]
Posting a note that I will host on the 25th of August as well.

**Comment by chaion07** [comment]
Requesting advice on the logistics such as Zoom Meeting details for the coffee break facilitation on 11 August. Thanks in advance!

**Comment by chaion07** [comment]
Happy to facilitate the coffee break for 8 September 2022

**Comment by nalininsbs** [comment]
I have updated the handbook page with the various updates that have been posted and removed the link descriptors using 'here' for accessibility. To assist potential volunteers, we can include a link to the page in recruitment. It is a working guide, and if anyone has an additional suggestion to be included which would assist volunteers facilitating, add to this GitHub card. Thank you.

**Comment by nalininsbs** [comment]
Thank you @chaion07 for volunteering. Will you be using Zoom or Google Meet? We can include it in the meeting on the previous day.
 
 I recall someone else coming forward for one of the future coffee breaks at the last or previous meeting to be added to the card.

**Comment by chaion07** [comment]
Hi, @nalininsbs! Thank you for the confirmation. I will be using Zoom (with a 40 minute limit per call).

**Comment by jpantani** [comment]
I can facilitate the November 17th coffee break and intend to use this Google Meet Link

**Comment by jpantani** [comment]
I can facilitate the November 17th coffee break and intend to use this Google Meet Link:
Coffee break placeholder
Thursday, November 17 · 1:30 – 2:15pm
Google Meet joining info
Video call link: https://meet.google.com/ipi-zyac-fpx
Or dial: ‪(CA) +1 437-781-4585‬ PIN: ‪812 307 896 0344‬#
More phone numbers: https://tel.meet/ipi-zyac-fpx?pin=8123078960344

**Comment by robinwpdeveloper** [comment]
`3 November 2022 - @robin and @eidolonnight`

I will be co hosting on November 3rd.
Please update my username to @robinwpdeveloper

**Comment by abhansnuk** [comment]
Adding 2023 team reps to this card: @robinwpdeveloper @sereedmedia @santanainniss

**Comment by robinwpdeveloper** [comment]
Hi @abhansnuk 
I have one question. 
How do we update the facilitator list?

Is it via new comment or updating the ticket description?
Currently I am unable to edit this ticket's description.

**Comment by eidolonnight** [comment]
Invite to the Make Marketing Github team sent. You should be able to edit this issue.

**Comment by becreativemkt** [comment]
Hi guys, if we can alternate times, I could take one at 11 AM UK time or 4 PM UK.

(18:30 is dinner time and I have to help with my dragons)

**Comment by becreativemkt** [comment]
23 feb count me in! 

My availability: I could take one at 11 AM UK time or 2-4 PM UK. (ending 5PM UK as latest)

**Comment by sereedmedia** [comment]
This issue should be closed to create individual cards for each session.

**Comment by chaion07** [comment]
> This issue should be closed to create individual cards for each session.

Thanks @sereedmedia I second this and available to host a coffee break in the near future. Thanks!

**Comment by eidolonnight** [comment]
> This issue should be closed to create individual cards for each session.

I agree. Do we have a tag for these that we can link to before closing this?

**Comment by nalininsbs** [comment]
We were using 'collaboration' for these and this may be suitable for the new coffee break ideas going forward too.
